candig/server/frontend.py

```diff
@@ -623,24 +632,20 @@ def handleAccessPermission(self):
 
         if app.config.get("TYK_ENABLED"):
             if 'Authorization' in self.request_dict.headers:
-                access_map = {}
-                token = self.request_dict.headers['Authorization']
-                parsed_payload = _parseTokenPayload(token)
-
+                access_token = self.request_dict.headers['Authorization']
+                parsed_payload = _parseTokenPayload(access_token)
+                issuer = parsed_payload.get('iss')
```

Comment:
Can we set a default value in case "iss" is not in the payload

Reply:
If `iss` isn't in the payload, is it not default to None?

Reply:
default is None for .get(). Do you see a place where that may cause an issue?

Reply:
you are right, getUserAccessMap would handle it.